docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,74 @@
+* Makes various UI improvements to the alert details flyout ({pull}170279[#170279], {pull}169035[#169035], {pull}173399[#173399], {pull}170078[#170078], {pull}168297[#168297]).
+* Saves the state of the alert details flyout in the browser. For example, after you use the flyout's *Investigate in timeline* button, you can click your browser's back button to return to the flyout ({pull}169661[#169661]).
+* Integrates a new Event Tracing for Windows (ETW) provider (Microsoft-Windows-Win32k) to create new event types that can be used by prebuilt endpoint rules to detect keylogging activity.
+* Allows for acting and target memory region buffers within behavior alerts to be scanned against {elastic-sec}'s collection of YARA signatures when collected. Detections are added to alerts.
+* Adds a new ReadProcessMemory (lsass) event that can be used by prebuilt endpoint rules to detect credential dumping.
+* Adds a link to the Edit connector UI for Amazon Bedrock that goes to the token tracking dashboard ({pull}172115[#172115]).
+* Allows you to use the `matches` operator when defining Endpoint rule exceptions and trusted applications.
+
+[discrete]
+[[bug-fixes-8.12.0]]
+==== Bug fixes
+
+* Fixes response action bugs by mapping the `unisolate` command to the `release` command and the `running-processes` command to the `processes` command ({pull}173831[#173831]).
+* Fixes the dark theme for the alert details flyout footer ({pull}173577[#173577]).
+* Makes the Timeline tour compatible with the Timeline template page ({pull}173526[#173526]).
+* Stops the {esql} tab from rendering when Timeline isn't open ({pull}173484[#173484]).
+* Fixes a bug that caused {ml} fetch jobs to fail when the default data view (`securitySolution:defaultIndex`) contained special characters ({pull}173426[#173426]).
+* Remove the **Assignees** field from the event details flyout ({pull}173314[#173314]).
+* Fixes a bug that caused the **Add to Case** action to fail if you didn't add a comment before isolating and releasing a host ({pull}172912[#172912]).
+* Fixes a UI bug that overlaid **Default Risk score** values as you created a new rule ({pull}172677[#172677]).
+* Fixes a bug that cleared configured fields in the exceptions flyout after the flyout reloaded and refocused ({pull}172666[#172666]).
+* Limits the character length for exception comments to 3000 characters, and makes the error message more descriptive if the limit's exceeded ({pull}170764[#170764]).
+* Adds missing alerts index filtration from sourcerer that was removed during refactoring ({pull}170484[#170484]).
+* Fixes a bug that didn't allow exceptions to be created or edited after an error displayed ({pull}169801[#169801]).
+* Stops {security-app} pages from crashing when there's a fields error in the **Stack by** component ({pull}168411[#168411]).
+* Adds a feature flag (`timelineEsqlTabDisabled`) for the **{esql}** tab in Timeline ({pull}174029[#174029]).
```

Comment:
should we add the emtpy query for the esql tab as well?

Reply:
and also maybe the fact that we do not render the esql tab unless visited?

Reply:
Resolved in [d22b48b](https://github.com/elastic/security-docs/pull/4469/commits/d22b48bb253a451a27bf1688ca5d5869e9406206).